deepinv/models/aliasfree.py

```diff
@@ -0,0 +1,740 @@
+# https://github.com/hmichaeli/alias_free_convnets
```

Comment:
Could you add a note saying what code is directly from here, what code is adapted from here, and what code is our original code?

Reply:
esp with the classes with no documentation - if they're not our implementations that's ok but if they are they should be properly documented